WinUi3Template1/MainWindow.xaml.cs

```diff
@@ -12,29 +18,28 @@ public MainWindow()
             Title = Settings.FeatureName;
 
             var hwnd = WinRT.Interop.WindowNative.GetWindowHandle(this);
-            LoadIcon(hwnd,"Assets/windows-sdk.ico");
+            LoadIcon(hwnd, "Assets/windows-sdk.ico");
             SetWindowSize(hwnd, 1920, 1080);
         }
 
         private void LoadIcon(IntPtr hwnd, string iconName)
         {
+            var hIcon = LoadImage(null, iconName, GDI_IMAGE_TYPE.IMAGE_ICON, 16, 16, IMAGE_FLAGS.LR_LOADFROMFILE);
```

Comment:
nit: as a code reviewer I can't see the inferred type of `hIcon`, which makes reviewing harder. A nice rule is if the type ain't obvious, don't use `var`.